server/server.go

```diff
@@ -83,6 +83,8 @@ func setupRoutes() {
 func StartLocalServer(port int) {
 	addr := fmt.Sprintf("0.0.0.0:%v", port)
 	log.Printf("Running server at %v\n", addr)
+    fs := http.FileServer(http.Dir("dist"))
```

Comment:
We may want to look into creating our own custom FileServer. Some websites warn against using the default straight out of the box? We can look into this later